configure.ac

```diff
@@ -1378,15 +1378,15 @@ fi
 
 
 # DECODED CERT CACHE
-AC_ARG_ENABLE([decertcache],
-    [AS_HELP_STRING([--enable-decertcache],[Enable decoded cert cacheing (default: disabled)])],
-    [ ENABLED_DECERTCACHE=$enableval ],
-    [ ENABLED_DECERTCACHE=no ]
+AC_ARG_ENABLE([setcertcache],
```

Comment:
Thought about this some more and this feature only applies to certgen... `defined(WOLFSSL_CERT_EXT) && defined(WOLFSSL_CERT_GEN)`. So perhaps `--enable-certgencache` is the right option with define `WOLFSSL_CERT_GEN_CACHE`?